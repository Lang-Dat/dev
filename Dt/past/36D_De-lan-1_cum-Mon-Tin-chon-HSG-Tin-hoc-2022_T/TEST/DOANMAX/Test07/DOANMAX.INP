23
-24213 22034 15054 -15503 3802 14690 16328 -3688 6629 24506 5167 -25982 5411 -824 8219 -10768 6764 -520 7390 -17637 19290 -12573 -11664 