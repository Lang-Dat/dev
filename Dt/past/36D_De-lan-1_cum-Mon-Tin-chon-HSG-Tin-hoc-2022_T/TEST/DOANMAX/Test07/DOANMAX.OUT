4
-15503 3802 14690 16328